api/libproject/devstart.rb

```diff
@@ -281,46 +288,6 @@ def drop_cloud_cdr(*args)
   end
 end
 
-def connect_to_cloud_db(*args)
-  common = Common.new
-  op = WbOptionsParser.new("connect-to-cloud-db", args)
-  gcc = GcloudContextV2.new(op)
-  env = Workbench::read_vars(common.capture_stdout(%W{
-    docker-compose run --rm api gsutil cat gs://#{gcc.project}-credentials/vars.env
-  }))
-  CloudSqlProxyContext.new(gcc).run do
-    password = env["WORKBENCH_DB_PASSWORD"]
-    common.run_inline %W{
-      docker-compose run --rm mysql-cloud --user=#{env["WORKBENCH_DB_USER"]}
-      --database=#{env["DB_NAME"]} --password=#{password}},
-      redact=password
-  end
-end
-
-def update_cloud_config(*args)
```

Comment:
I don't think these commands should be run by developers, so I moved them into deploy so they are not displayed in the command list.

Reply:
These both need to be runnable by developers. Please move them back.

Reply:
👍